snap/implicit.go

```diff
@@ -53,11 +53,11 @@ var implicitSlots = []string{
 	"timezone-control",
 	"tpm",
 	"kernel-module-control",
+	"camera",
```

Comment:
+1 but can it be in alphabetical order? I see that kernel-module-control is out of order too :\